src/app/components/HallOfFame/index.js

```diff
@@ -0,0 +1,87 @@
+export class HallOfFame {
+  constructor(element) {
+    this.element = element;
+  }
+  display() {
+    this.element.innerHTML = `
+    <div class="mode__hall mode__rules">
+            <p class="mode__title">
+              <i class="fas fa-id-badge"></i> Mode ranking
+            </p>
+            <table class="ranking__table">
+              <h2 class="ranking__invitation">Our hall of fame is waiting for your results!</h2>
+              <tr class="ranking__header">
+                <th class="ranking__header--bold">Place</th>
+                <th class="ranking__header--bold">Player</th>
+                <th class="ranking__header--bold">Answered</th>
+              </tr>
+              <tr class="ranking__row ranking__row--first">
+                <td class="ranking__place ">1st</td>
+                <td class="ranking__name">Ania</td>
+                <td class="ranking__score">15/20</td>
+              </tr>
+              <tr class="ranking__row ranking__row--second">
+                <td class="ranking__place">2nd</td>
+                <td class="ranking__name">Mateusz</td>
+                <td class="ranking__score">14/30</td>
+              </tr>
+              <tr class="ranking__row ranking__row--third">
+                <td class="ranking__place">3rd</td>
+                <td class="ranking__name">Leia Organa</td>
+                <td class="ranking__score">1/23</td>
+              </tr>
+            </table>
+          </div>
+        </div>
+      </div>`;
+  }
+  hide() {
+    this.element.innerHTML = '';
+  }
+  saveResult(result) {
+    // podział na game type
+    let resultStorage = this.getBestResults();
+    resultStorage.unshift(result);
+    resultStorage = resultStorage
+      .sort((a, b) => {
+        // logika najlepszego wyniku
```

Comment:
miejsca gdzie dopisujesz komentarz dla czytelnosci to dobre miejsce na wydzielenie metody o odpowiedniej nazwie :) 
np. sortScores/result i przekazanie jako parametr do saveResult

Reply:
Ja jeszcze dorzucę, że jeśli mamy jedna linijkę to możemy pominąć klamry i return, więc kod się skróci o 2 linie 😄 
https://riptutorial.com/javascript/example/17665/implicit-return

Reply:
tam akurat będzie dopisany kod, więc nie będzie okazji do skrótów, ale przyjrzę się, gdzie da się zoptymalizować ;)